config/config.json

```diff
@@ -1,22 +1,22 @@
 {
   "development": {
-    "username": "bp",
+    "username": "jamescape",
```

Comment:
Let's try to get dotenv or another fix for this so we don't have to change everytime we pull to our local branch.